corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/entities/app_selection.js

```diff
@@ -0,0 +1,60 @@
+FormplayerFrontend.module("Entities", function (Entities, FormplayerFrontend, Backbone, Marionette, $, _) {
+    Entities.AppSelect = Backbone.Model.extend({
+        urlRoot: "appSelects"
+    });
+
+    Entities.configureStorage("FormplayerFrontend.Entities.AppSelect");
+
+    Entities.AppSelectCollection = Backbone.Collection.extend({
+        url: "appSelects",
+        model: Entities.AppSelect
+    });
+
+    Entities.configureStorage("FormplayerFrontend.Entities.AppSelectCollection");
```

Comment:
Need a way to store the list of apps that HQ hands off to this application. Currently doing this in the local browser storage - open to suggestions if there's a better way